# set_modified was stablilized in Rust 1.75, discard this patch once we've updated
diff --git a/src/tunnel/tls_reloader.rs b/src/tunnel/tls_reloader.rs
index 47ae5709bce..e08ab9d4bef 100644
--- a/src/tunnel/tls_reloader.rs
+++ b/src/tunnel/tls_reloader.rs
@@ -86,10 +86,6 @@ impl TlsReloader {
             let Ok(file) = File::open(&path) else {
                 return;
             };
-            let _ = file.set_modified(SystemTime::now()).map_err(|err| {
-                error!("Cannot force reload TLS file {:?}:  {:?}", path, err);
-                error!("Old certificate will be used until the next change");
-            });
         });
     }
 
